examples/miral-shell/shell_main.cpp

```diff
@@ -132,24 +139,35 @@ int main(int argc, char const* argv[])
       }
     };
 
-
-    return runner.run_with(
-        {
-            CursorTheme{"default:DMZ-White"},
-            WaylandExtensions{},
-            X11Support{},
-            ConfigureDecorations{},
-            window_managers,
-            display_configuration_options,
-            external_client_launcher,
-            launcher,
-            config_keymap,
-            AppendEventFilter{quit_on_ctrl_alt_bksp},
-            StartupInternalClient{spinner},
-            ConfigurationOption{run_startup_apps, "startup-apps", "Colon separated list of startup apps", ""},
-            pre_init(ConfigurationOption{[&](std::string const& typeface) { ::wallpaper::font_file(typeface); },
-                                         "shell-wallpaper-font", "font file to use for wallpaper", ::wallpaper::font_file()}),
-            ConfigurationOption{[&](std::string const& cmd) { terminal_cmd = cmd; },
-                                "shell-terminal-emulator", "terminal emulator to use", terminal_cmd}
-        });
+    return runner.run_with({
+        CustomDecorations{UserDecoration::create_manager},
+        CursorTheme{"default:DMZ-White"},
+        WaylandExtensions{},
+        X11Support{},
+        ConfigureDecorations{},
+        window_managers,
+        display_configuration_options,
+        external_client_launcher,
+        launcher,
+        config_keymap,
+        AppendEventFilter{quit_on_ctrl_alt_bksp},
+        StartupInternalClient{spinner},
+        ConfigurationOption{run_startup_apps, "startup-apps", "Colon separated list of startup apps", ""},
+        pre_init(ConfigurationOption{
+            [&](std::string const& typeface)
+            {
+                ::wallpaper::font_file(typeface);
+            },
+            "shell-wallpaper-font",
+            "font file to use for wallpaper",
+            ::wallpaper::font_file()}),
+        ConfigurationOption{
+            [&](std::string const& cmd)
+            {
+                terminal_cmd = cmd;
+            },
+            "shell-terminal-emulator",
+            "terminal emulator to use",
+            terminal_cmd},
+    });
```

Comment:
I *think* the only substantive change is adding `CustomDecorations{UserDecoration::create_manager},`. The rest is unnecessary, and distracting, whitespace adjustment